static/templates/actors/hazard/partials/sidebar.html

```diff
@@ -89,7 +89,7 @@ <h2>{{localize "PF2E.ImmunitiesLabel"}}</h2>
             </div>
             <div class="tags">
                 {{#each immunities as |immunity slug|}}
-                    <span class="tag" data-slug="{{slug}}">{{localize immunity.label}}</span>
```

Comment:
I think workshop uses that?

Reply:
Does it? The weakness/resistance ones for hazard were not right at all, and NPC does data-weakness (for all of them....). Should probably make them all the same then. @xdy do you use these slugs, and if so, which one do you use? data-weakness?

Reply:
Maybe you can fix 🙂 
https://github.com/foundryvtt/pf2e/issues/3657

Reply:
I'll wait to see what the response is, but I'm leaning towards data-slug and data-type.

Reply:
No, I do it in a createChatMessage hook on a damageroll and then check message.flags?.pf2e?.damageRoll?.types for the attack (or if that's empty, get the original item and check damage.value and get the types) and then compare to traits?.di/dr/dv.value on the target. (In a very messy way. Total rewrite needed.)

Reply:
Sorry, we got you mixed up. Up to @shemetz then, but I'm likely going to at least be adding data-slug. Renaming data-weakness might be too annoying to do right now though?

Reply:
Changed it to data-weakness="slug", data-resistance="slug", and data-immunity="slug".

Reply:
Thanks for notifying me!